.github/workflows/deploy-issuer-func.yaml

```
@@ -6,11 +6,15 @@ on:
 jobs:
   deploy_workspace_to_azure:
     name: Deploy
-    uses: pagopa/io-std/.github/workflows/deploy-workspace.yaml@main
-    with:
-      environment: io-p-sign-issuer-func
-      workspace-name: io-func-sign-issuer
+    uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@main
     secrets: inherit
+    with:
+      workspace_name: io-func-sign-issuer
+      environment: webapp-prod-cd
```

```suggestion
      environment: webapp-prod
```

`cd` is appended by the template